selfdrive/test/process_replay/regen.py

```diff
@@ -127,7 +127,10 @@ def replay_cameras(lr, frs, disable_tqdm=False):
   ]
 
   def replay_camera(s, stream, dt, vipc_server, frames, size, use_extra_client):
-    pm = messaging.PubMaster([s, ])
+    services = [(s, stream)]
+    if use_extra_client:
+      services.append(("wideRoadCameraState", VisionStreamType.VISION_STREAM_WIDE_ROAD))
+    pm = messaging.PubMaster([s for s, _ in services])
```

Comment:
Not the cleanest, I'd like to fix why it takes so long to start the first cam proc